pkg/controller/unifiedpushserver/unifiedpushserver.go

```diff
@@ -64,7 +65,31 @@ func newOauthProxyRoute(cr *pushv1alpha1.UnifiedPushServer) (*routev1.Route, err
 	}, nil
 }
 
-func newUnifiedPushServerDeployment(cr *pushv1alpha1.UnifiedPushServer) (*appsv1.Deployment, error) {
+func newOauthProxyImageStream(cr *pushv1alpha1.UnifiedPushServer) (*imagev1.ImageStream, error) {
```

Comment:
Like PostgreSQL, there's also a default ImageStream for the oauth-proxy available through the `openshift` namespace, so maybe we could/should use that.

@maleck13 note that for the default one in this case in the openshift project, the only tag (`v1.1.0`) has `importPolicy.scheduled = true` (postgresql doesn't have this).

Reply:
@grdryn I don't see that on Minikube. You sure about this?

![Screenshot from 2019-06-25 10-45-52](https://user-images.githubusercontent.com/376732/60079351-70eb2e80-9736-11e9-9554-c234d64b0d0c.png)

Reply:
Huh, that's interesting, it's there when I look on OCP/OSD. I guess it's fine to create one then. Given that that ImageStream would just be used just for UPS, I think it might actually be a good thing for it to have the `importPolicy.scheduled = true`, so that we don't have to instruct operations people that when they `oc import-image` for their other ImageStreams that they need to remember to do this one also.

Reply:
Ok, so 
OauthProxy --> scheduled=true
UPS --> scheduled=false

Is this what you think?

Reply:
Ugh, that does sound insane so maybe not. I guess set them both to false based on @maleck13's previous guidance.

@aliok do you have any thoughts on what they should be set to? I have to admit to not really liking the inclusion of imagestreams in the first place, so I'm just trying to make sure we actually satisfy the requirements from those that have foisted it upon us. I guess because UPS is our responsibility though, we have some say in the matter: do we want clusters to update their UPS deployments automatically when there's a new UPS or oauth-proxy image in the registry, or should they only do that in a maintenance window?

Reply:
IMO, our SRE's should decide on this. I was trying to get their opinion on similar matters with an email on AeroGear ML but I guess I wasn't successful. Let's do `scheduled=false` for now for both then.